src/index.ts

```diff
@@ -47,20 +51,26 @@ export async function startNotarize(opts: NotarizeStartOptions): Promise<Notariz
     }
     d('zip succeeded, attempting to upload to apple');
 
+    const notarizeStartOpts = [
+      'altool',
+      '--notarize-app',
+      '-f',
+      zipPath,
+      '--primary-bundle-id',
+      opts.appBundleId,
+      '-u',
+      makeSecret(opts.appleId),
+      '-p',
+      makeSecret(opts.appleIdPassword),
+    ];
+
+    if (opts.ascProvider !== '') {
+      notarizeStartOpts.push(...['-itc_provider', opts.ascProvider]);
```

Comment:
You don't need to spread an array here, you can just call it with multiple args
```suggestion
      notarizeStartOpts.push('-itc_provider', opts.ascProvider);
```